apps/artemis-web/utils/parsers.tsx

```diff
@@ -312,7 +312,7 @@ function extractHijackInfoLeft(
 function extractHijackInfoRight(hijack) {
   const hijackInfo = {
     'Time Started': [
-      formatDate(new Date(hijack.time_started), 2),
+      formatDate(new Date(hijack.time_started), Math.abs(new Date().getTimezoneOffset() / 60)),
```

Comment:
please also do the same for mitigation started (not sure why this is not implemented there), and also check again the 'Never' alternative logic in timestamps. In some of them we fall back to 'Never', in others no. All fixes pertain to the `parsers.tsx`. Thanks @CuriouzK0d3r !

Reply:
Nice catch! fixed!